charts/ingress-nginx/values.yaml

```diff
@@ -486,12 +486,12 @@ controller:
     external:
       # -- Enable the external controller service or not. Useful for internal-only deployments.
       enabled: true
-    # -- Annotations to be added to the external controller service. See `controller.service.internal.annotations` for annotations to be added to the internal controller service.
-    annotations: {}
-    # -- Labels to be added to both controller services.
-    labels: {}
-    # -- Type of the external controller service.
-    # Ref: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
+      # -- Annotations to be added to the external controller service. See `controller.service.internal.annotations` for annotations to be added to the internal controller service.
+      annotations: {}
+      # -- Additional custom labels specifically for the external controller service
+      labels: {}
+      # -- Type of the external controller service.
+      # Ref: https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
```

Comment:
Please do not remove the existing values. Only add your `labels` to the `external` object right below the `enabled` value.

Reply:
Understood, done.